.github/workflows/checks.yml

```diff
@@ -28,3 +28,29 @@ jobs:
       - run: yarn
       - run: yarn run prettify:diff
       - run: yarn run lint
+  mobile-tests:
+    name: Mobile Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+          check-latest: true
+      - name: Install Yarn dependencies
+        run: yarn
+      - name: Build Yarn packages
+        run: yarn build --ignore docs
+      - name: Verify locales if using l10n/* branch
```

Comment:
Totally orthogonal to this PR: why is test:verify-locales part of this job vs running in it's own standalone job?

Reply:
Fair point! I'll break it out into its own.

Reply:
It fails but because the current branch doesn't have all of the updated locales. https://github.com/celo-org/wallet/runs/2889108770?check_suite_focus=true. We can test again closer to the release to confirm it works as intended but it is the same script as what ran on Circle.